infra/aws/terraform/cdn.tf

```diff
@@ -38,48 +38,52 @@ resource "aws_cloudfront_distribution" "this" {
     }
   }
 
-  origin {
-    domain_name = format("s3-content.%s.%s.vittle.ai.s3.amazonaws.com", var.region, var.environment)
-    origin_path = "/static-content"
-    origin_id   = format("S3-content.%s.%s.vittle.ai/static-content", var.region, var.environment)
-  }
-
-  price_class = "PriceClass_All"
-
   restrictions {
     geo_restriction {
       restriction_type = "none"
     }
   }
+
+  default_cache_behavior {
+    allowed_methods        = ["GET", "HEAD"]
+    cache_policy_id        = "658327ea-f89d-4fab-a63d-7e88639e58f6"
```

Comment:
I suppose this is ID of some managed policy, can you use the datasource instead?
eg
```
data "aws_cloudfront_cache_policy" "this" {
  name = "Managed-CachingOptimized"
}
...
 cache_policy_id          = data.aws_cloudfront_cache_policy.this.id
```
?

Reply:
I'd rather use the `id` as it's not subject to changes as a name can be. However, I understand it's not easy to read and it makes you go to the AWS docs to know about that policy or at least add a comment to state which policy you're using.

What path do you prefer?

Reply:
well, magic string is in most cases less preferable than data source :) if possible please use the data source instead

Reply:
ok, I'll go that way